resources/views/home/_search_forum_post.blade.php

```diff
@@ -15,19 +15,47 @@
     You should have received a copy of the GNU Affero General Public License
     along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 --}}
-<a
-    class="search-entry"
-    href="{{ post_url($entry->topic_id, $entry->getKey()) }}"
->
-    <h1 class="search-entry__row search-entry__row--title">
-        {{ $entry->topic === null ? '['.trans('forum.topic.deleted').']' : $entry->topic->topic_title }}
-    </h1>
 
-    <p class="search-entry__row search-entry__row--excerpt">
-        {{ html_excerpt($entry->body_html) }}
-    </p>
+@foreach ($results as $index => $hit)
+    @php
+        $source = $hit['_source'];
+        $innerHits = $results->innerHits($index, 'posts');
+        $firstPost = $results->innerHits($index, 'first_post');
+        $firstPostUrl = route('forum.topics.show', $source['topic_id']);
+    @endphp
+    <div class="search-result__entry search-result__entry--threaded">
```

Comment:
the original design (?) is each blade file doesn't contain element which block located in different file ಠ_ಠ